app/models/event.rb

```diff
@@ -30,6 +27,25 @@ def self.location_info(location)
     location.name
   end
 
+  def location
```

Comment:
Just a personal choice but I would normally go for a one liner as location_barcode is never used for anything else except for finding the location. Also is there a reason for the begin?

Reply:
Sometimes I like to have the variable just for readability, but I think you're right here it will be clear enough as a one liner.
I assumed the begin / end was needed because it's a multi-line block - what would the alternative be (if keeping the two lines)?
Thanks